lib/cc/cli/help.rb

```diff
@@ -14,7 +14,7 @@ def run
 
       def commands
         [
-          "analyze [-f format]",
+          "analyze [-f format] [-e engine] <path>",
```

Comment:
I don't think it needs to be removed, but want to mention that this seems like a drive-by fix